### ISSUE TITLE
properties are not publicly re-exported

### ISSUE THREAD

**Issue by mavdotj** [radu781/dualsense-rs#2]
tried to do a match on the symbols but I can't import Symbols because it's not re-exported in lib.rs

**Comment by mavdotj** [comment]
```
error[E0603]: module `properties` is private
  --> src/main.rs:2:19
   |
2  | use dualsense_rs::properties::symbols::Symbols;
   |                   ^^^^^^^^^^           ------- enum `Symbols` is not publicly re-exported
   |                   |
   |                   private module
   |
note: the module `properties` is defined here
  --> (cargo registry)/dualsense-rs/src/lib.rs:34:9
   |
34 | pub use dualsense::*;
   |  
   ```

**Comment by radu781** [comment]
Fixed in 5e2aeb455bf148a246b8d069fa343a1af4f9067f, did not create a pull request to address only this issue because it was difficult to separate the needed changes. Thanks for the interest in this project!
